scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -399,16 +402,14 @@
 
 (defn rebalance
   "Takes a db, a list of pending job entities, a map of spare resources and params.
-   Returns a list of pending job entities to run and a list of task entities to preempt
-
-   category is :normal or :gpu, depending on which type of job we're working with"
+   Returns a list of pending job entities to run and a list of task entities to preempt"
   [db agent-attributes-cache pending-job-ents host->spare-resources rebalancer-reservation-atom
-   {:keys [max-preemption category] :as params}]
+   {:keys [max-preemption pool-ent] :as params}]
   (let [timer (timers/start rebalance-duration)
         jobs-to-make-room-for (->> pending-job-ents
                                    (filter (partial util/job-allowed-to-start? db))
                                    (take max-preemption))
-        init-state (init-state db (util/get-running-task-ents db) jobs-to-make-room-for host->spare-resources category)
+        init-state (init-state db (util/get-running-task-ents db) jobs-to-make-room-for host->spare-resources pool-ent)
```

Comment:
I'm not sure. Why is it better for the rebalancer to not have any pool-specific concerns?

Reply:
Simplifies the function by reducing its responsibility at the cost of someone higher up performing some of the work. E.g. its for the same reason we pass in `pending-job-ents` instead of passing the map and asking the function to lookup the pending jobs using the pool.

Reply:
I misread your suggestion, you can ignore the above comment from me.

Done.